docs/it/ciclo-del-feedback.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: default
+title: Ciclo del Feedback
+nav_order: 999
+---
+
+# Il Ciclo del Feedback
+
+## La flessibilità del software
+
+Intuitivamente parlando, sembra ovvio che noi sviluppatori, con una connessione internet e un portatile alla mano, possiamo essere enormemente più produttivi di decine di operai da catena di montaggio del secolo scorso. Ma perché è davvero così?
+
+Sicuramente è grazie al fatto che possiamo lavorare con strumenti frutto di decenni di evoluzione tecnologia esponenziale, ma la vera ragione, il vero discriminante, sta nel fatto nel "soft" in "software": il prodotto del nostro lavoro è spesso intangibile e facilmente manipolabile, tanto che se commettiamo un errore, non dobbiamo prendere il semilavorato e buttarlo via, sprecando materie prime, ma nella peggiore delle ipotesi abbiamo solo sprecato un po' di tempo.
+
+Tutto questo non è nient'altro che la "Terza rivoluzione industriale", come viene chiamata nei libri di scuola; dopo alcuni decenni in cui il collo di bottiglia era la potenza di calcolo, al giorno d'oggi abbiamo a disposizione CPU e memoria sufficienti per qualunque task mondano, e questa pura forza bruta si schianta ora invece contro problemi organizzativi: far lavorare decine, se non centinaia di sviluppatori assieme sullo stesso progetto software è un problema non banale, e che tanti prima di noi si sono trovati a dover risolvere.
+
+Uno strumento che è emerso vincitore da queste difficoltà è il "ciclo del feedback"; ma di che si tratta? E' un approccio in cui, in seguito di una semplice azione, si va a misurarne rapidamente l'effetto (il "feedback"), così da ottenerne nuove informazioni, che saranno impiegate per intraprendere i prossimi passi in maniera più conscia e informata, creando così il "ciclo". Questo ciclo generalmente permette di procedere in una attività a piccoli passi ma con una discreta rapidità e, cosa ancora più importante, sempre nella giusta direzione, visto che eventuali errori di rotta possono essere rapidamente corretti ad ogni passo.
+
+## Il manifesto Agile e l'Extreme Programming
+
+Alcuni movimenti che sono nati in seno alle difficoltà di questa rivoluzione sono il Manifesto Agile, l'Extreme Programming, il DevOps: il primo ad alto livello, il secondo ad un livello più "basso" tecnico, il terzo in maniera "mista", hanno tutti cercato di trovare modi per scrivere software in maniera più snella e organizzata, in modo che non ci si ritrovasse più in una melassa di linee di codice che, dopo pochi mesi di lavoro, diventava ingestibile.
```

Comment:
```suggestion
Alcuni movimenti che sono nati in seno alle difficoltà di questa rivoluzione sono il Manifesto Agile, l'Extreme Programming (XP), il DevOps: il primo ad alto livello (_descrittivo_), il secondo ad un livello più _basso_ (tecnico), il terzo di tipo "misto", hanno tutti cercato modi per scrivere software in maniera più snella e organizzata, in modo che non ci si ritrovasse più in una melassa di linee di codice che, dopo pochi mesi di lavoro, risultasse ingestibile.
```

Reply:
L'ho provata a correggere qua e là ma, pur comprendendo il senso che vuoi dare al contenuto, la proverei a riscrivere. Mi sembra un periodo troppo lungo in cui ci sono tanti concetti che al lettore potrebbero non essere noti. Che ne dici di prendere più respiro e aprire ognuno di questi 3 movimenti con qualche accenno o nozione aggiuntiva?

Inoltre attenzione che se la sezione la chiamiamo "Il manifesto Agile e l'Extreme Programming" e poi parli anche di DevOps rischiamo di confondere il lettore.